spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -1326,7 +1326,7 @@ public void testParallelPartialProgressWithMaxFailedCommits() {
   }
 
   @TestTemplate
-  public void testParallelPartialProgressWithMaxFailedCommitsLargerThanTotalFileGroup() {
+  public void testParallelPartialProgressWithMaxCommitsLargerThanTotalFileGroup() {
```

Comment:
Since you are renaming the test, may I suggest `testParallelPartialProgressWithMaxCommitsGreaterThanTotalGroupCount`? as the number returned by `FileRewritePlan` is called `totalGroupCount`.